src/media/providers/exifr/exifr-provider.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { UploadFileDto } from '../../dto/file';
+import { GeoReferenceProvider } from '../types/geo-reference-provider';
+import { Point } from 'geojson';
+import exifr from 'exifr';
+
+@Injectable()
+export class ExifrProvider implements GeoReferenceProvider {
+  async extractGeoLocation(file: UploadFileDto): Promise<Point | null> {
+    const { buffer } = file;
+    const coordinates = await exifr.gps(buffer);
+
+    if (coordinates) {
+      return this.getPoint(coordinates.latitude, coordinates.longitude);
```

Comment:
What happens if the coordinates are somewhere in the states? Will this just affect the displaying of the coordinates in the frontend?

Reply:
@Saela Yes, the points will just not be visible. That's only an issue because we're currently using swisstopo maps and restricting our extent, something we'll likely change soon.